commit a6a50ead595a73a5a021e98b12d1ebdacf5fac87
Author: RahifM <rahifmanjatha372@gmail.com>
Date:   Fri Aug 19 05:10:00 2022 +0000

    Revert "vendor: Add qcom platform type to soongs vars"
    
    This reverts commit 97cb87c80972e109dedf58fb71b03489b97dd923.

diff --git a/build/soong/android/variable.go b/build/soong/android/variable.go
index 2136f01e..81e2a94a 100644
--- a/build/soong/android/variable.go
+++ b/build/soong/android/variable.go
@@ -51,26 +51,6 @@ type Product_variables struct {
 		Cppflags []string
 		Shared_libs []string
 	}
-	Uses_qcom_um_family struct {
-		Cflags []string
-		Srcs []string
-	}
-	Uses_qcom_um_3_18_family struct {
-		Header_libs []string
-		Shared_libs []string
-	}
-	Uses_qcom_um_4_4_family struct {
-		Header_libs []string
-		Shared_libs []string
-	}
-	Uses_qcom_um_4_9_family struct {
-		Header_libs []string
-		Shared_libs []string
-	}
-	Uses_qcom_um_4_14_family struct {
-		Header_libs []string
-		Shared_libs []string
-	}
 }
 
 type ProductVariables struct {
@@ -92,9 +72,4 @@ type ProductVariables struct {
 	Uses_nvidia_enhancements  *bool `json:",omitempty"`
 	Uses_qcom_bsp_legacy  *bool `json:",omitempty"`
 	Uses_qti_camera_device  *bool `json:",omitempty"`
-	Uses_qcom_um_family  *bool `json:",omitempty"`
-	Uses_qcom_um_3_18_family  *bool `json:",omitempty"`
-	Uses_qcom_um_4_4_family  *bool `json:",omitempty"`
-	Uses_qcom_um_4_9_family  *bool `json:",omitempty"`
-	Uses_qcom_um_4_14_family  *bool `json:",omitempty"`
 }
diff --git a/build/soong/soong_config.mk b/build/soong/soong_config.mk
index 0820d67b..e7a1987a 100644
--- a/build/soong/soong_config.mk
+++ b/build/soong/soong_config.mk
@@ -22,11 +22,6 @@ $(call add_json_bool, Uses_metadata_as_fde_key, $(filter true,$(TARGET_USES_META
 $(call add_json_bool, Uses_nvidia_enhancements, $(filter true,$(NV_ANDROID_FRAMEWORK_ENHANCEMENTS)))
 $(call add_json_bool, Uses_qcom_bsp_legacy, $(filter true,$(TARGET_USES_QCOM_BSP_LEGACY)))
 $(call add_json_bool, Uses_qti_camera_device, $(filter true,$(TARGET_USES_QTI_CAMERA_DEVICE)))
-$(call add_json_bool, Uses_qcom_um_family, $(filter true,$(TARGET_USES_QCOM_UM_FAMILY)))
-$(call add_json_bool, Uses_qcom_um_3_18_family, $(filter true,$(TARGET_USES_QCOM_UM_3_18_FAMILY)))
-$(call add_json_bool, Uses_qcom_um_4_4_family, $(filter true,$(TARGET_USES_QCOM_UM_4_4_FAMILY)))
-$(call add_json_bool, Uses_qcom_um_4_9_family, $(filter true,$(TARGET_USES_QCOM_UM_4_9_FAMILY)))
-$(call add_json_bool, Uses_qcom_um_4_14_family, $(filter true,$(TARGET_USES_QCOM_UM_4_14_FAMILY)))
 
 # This causes the build system to strip out the last comma in our nested struct, to keep the JSON valid.
 _contents := $(_contents)__SV_END
diff --git a/config/BoardConfigQcom.mk b/config/BoardConfigQcom.mk
index d699edd8..0c65af2e 100644
--- a/config/BoardConfigQcom.mk
+++ b/config/BoardConfigQcom.mk
@@ -70,23 +70,15 @@ else ifneq ($(filter $(BR_FAMILY),$(TARGET_BOARD_PLATFORM)),)
 else ifneq ($(filter $(UM_3_18_FAMILY),$(TARGET_BOARD_PLATFORM)),)
     MSM_VIDC_TARGET_LIST := $(UM_3_18_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8996
-    TARGET_USES_QCOM_UM_FAMILY := true
-    TARGET_USES_QCOM_UM_3_18_FAMILY := true
 else ifneq ($(filter $(UM_4_4_FAMILY),$(TARGET_BOARD_PLATFORM)),)
     MSM_VIDC_TARGET_LIST := $(UM_4_4_FAMILY)
     QCOM_HARDWARE_VARIANT := msm8998
-    TARGET_USES_QCOM_UM_FAMILY := true
-    TARGET_USES_QCOM_UM_4_4_FAMILY := true
 else ifneq ($(filter $(UM_4_9_FAMILY),$(TARGET_BOARD_PLATFORM)),)
     MSM_VIDC_TARGET_LIST := $(UM_4_9_FAMILY)
     QCOM_HARDWARE_VARIANT := sdm845
-    TARGET_USES_QCOM_UM_FAMILY := true
-    TARGET_USES_QCOM_UM_4_9_FAMILY := true
 else ifneq ($(filter $(UM_4_14_FAMILY),$(TARGET_BOARD_PLATFORM)),)
     MSM_VIDC_TARGET_LIST := $(UM_4_14_FAMILY)
     QCOM_HARDWARE_VARIANT := sm8150
-    TARGET_USES_QCOM_UM_FAMILY := true
-    TARGET_USES_QCOM_UM_4_14_FAMILY := true
 else
     MSM_VIDC_TARGET_LIST := $(TARGET_BOARD_PLATFORM)
     QCOM_HARDWARE_VARIANT := $(TARGET_BOARD_PLATFORM)
